### PR TITLE
Added documentation/script for signing and pushing stable RPMs

### PR DESCRIPTION
**Pull request by hummeltech**
This can be used for other repositories as well (e.g. `geoint-deps`).

### CODE REVIEW

**Review by BuechnerGIS — APPROVED** [review]
:rocket:

**Review by jbronn — CHANGES_REQUESTED** [review]
As-is I'm -1.  From what I can tell, these scripts are signing *every* RPM that's built, even if their versions aren't changed.  I don't like our build artifacts changing each time we sign one release, in addition to the havoc this will wreak on Docker's BuildKit caching.

File: scripts/push_stable_rpms_from_zip.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+# Export COMPOSE_FILE variable
+export COMPOSE_FILE
+
+# Sign the extracted RPMs
+${DOCKER_RUN_CMD} \
+  bash -c "rpm --addsign EXTRACTED_RPMS/*/*.rpm"
+# Sync only the new (based on filename) extracted RPMs into the local repo copy
+${DOCKER_RUN_CMD} \
+  bash -c "rsync --archive --ignore-existing --verbose EXTRACTED_RPMS/*/*.rpm ${REPO_PREFIX}"
```
Comment (hummeltech) [inline scripts/push_stable_rpms_from_zip.sh:64]: @jbronn Yep, you must have missed this line, although all RPMs are indeed being signed, only the "new" ones are copied over.

Comment (jbronn) [reply scripts/push_stable_rpms_from_zip.sh:64]: Yes, I did miss this line and addresses my concerns.

**Review by jbronn — APPROVED** [review]
LGTM

**Pull request merged**